clients/java/src/main/java/io/camunda/zeebe/client/ZeebeClient.java

```diff
@@ -358,6 +359,40 @@ static ZeebeClientCloudBuilderStep1 newCloudClientBuilder() {
    */
   UpdateRetriesJobCommandStep1 newUpdateRetriesCommand(ActivatedJob job);
 
+  /**
+   * Command to update the timeout of a job.
+   *
+   * <pre>
+   * long jobKey = ..;
+   *
+   * zeebeClient
+   *  .newUpdateTimeoutCommand(jobKey)
+   *  .timeout(100)
+   *  .send();
+   * </pre>
+   *
+   * @param jobKey the key of the job to update
+   * @return a builder for the command
+   */
+  UpdateTimeoutJobCommandStep1 newUpdateTimeoutCommand(long jobKey);
+
+  /**
+   * Command to update the timeout of a job.
+   *
+   * <pre>
+   * ActivatedJob job= ..;
+   *
+   * zeebeClient
+   *  .newUpdateTimeoutCommand(job)
+   *  .timeout(100)
+   *  .send();
+   * </pre>
+   *
+   * @param job the activated job
+   * @return a builder for the command
+   */
```

Comment:
💭 It might be interesting for users to note any specifics here, e.g. does this add to the existing timeout or replace it, if it replaces it, does it start from now or from when the related command is processed?

Reply:
I think this is a very useful addition to the documentation, will update it now.

Reply:
Changes are ready for review.